gordo_components/cli/cli.py

```diff
@@ -124,6 +130,19 @@ def gordo():
     type=click.File(mode="w", lazy=False),
     default="/tmp/model-location.txt",
 )
+@click.option(
+    "--data-provider-threads",
```

Comment:
Some merge issues here I think, since this should be gone. But maybe you have not rebased on top of master?

Reply:
Yes, I have rebased on top of master recently, but obviously have made a mistake at some point. Have removed these now.